src/components/onCall/oncall-list-view.tsx

```diff
@@ -64,87 +72,123 @@ const OnCallListView = ({ selectedDate, setSelectedDate, updatePaidStatus }: Pro
       flex: 1,
       headerAlign: "center",
       align: "center",
-      sortable: true
+      sortable: true,
+      renderCell: (params) => (
+        <Typography
+          sx={{
+            fontWeight: isLoggedIn(params.row.email) ? "bold" : "normal",
+            border: isLoggedIn(params.row.email) ? "2px solid black" : "none",
+            borderRadius: 2,
+            px: 1
+          }}
+        >
+          {formatUsername(params.value)}
+        </Typography>
+      )
+    },
+    {
+      field: "email",
+      headerName: strings.userTable.email,
+      flex: 1,
+      headerAlign: "center",
+      align: "center",
+      sortable: true,
+      renderCell: (params) => (
+        <Typography
+          sx={{
+            fontWeight: isLoggedIn(params.value) ? "bold" : "normal"
+          }}
+        >
+          {params.value}
+        </Typography>
+      )
     }
   ];
 
   const rows = onCallData.map((item, idx) => ({
     id: idx,
     paid: item.paid,
     week: item.week,
-    person: item.username ? item.username : strings.oncall.noUsernameOnCall
+    person: item.username ? item.username : strings.oncall.noUsernameOnCall,
+    email: item.email ? item.email : "-"
```

Comment:
Ternary operator is ok here, but more typical and cleaner is the nullish coalescing operator
e.g.
item.username ?? strings.oncall.noUsernameOnCall
item.email ?? "-"